aten/src/ATen/native/Linear.cpp

```diff
@@ -18,9 +18,12 @@ Tensor bilinear(const Tensor& input1, const Tensor& input2, const Tensor& weight
   AT_ASSERT(input2.size(input2.dim() - 1) == weight.size(2),
             "bilinear(): input2 size does not match weight size: got %lld but expected %lld",
             (long long)input2.size(input2.dim() - 1), (long long)weight.size(2));
-  AT_ASSERT(bias.defined() && bias.size(0) == weight.size(0),
```

Comment:
I think someone just wanted to do `!bias.defined() || bias.size(0) == weight.size(0)`. Can you please use that expression to make the error checking code shorter?

Reply:
sure, I just thought it would be nicer to make it more explicit.